sdk/bare/common/drv/dac.c

```diff
@@ -1,3 +1,5 @@
+#if 0
```

Comment:
Sanity check:

Do you mean to be compiling this out?

If so, is there a reason we are keeping this code in here (but compiled out) as opposed to just removing it?

Reply:
The DAC code could still potentially work with the REV A DAC, but it would take someone some effort to get it running again (i.e. messing around in the FPGA, etc). I left in the code since Brandt could maybe use it to spring board his development. He will be replacing this driver to make it work with his REV B DAC.

So I would like to keep this code in the code base for the next few months until Brandt's project finishes up.

Reply:
👍